test/bntest.c

```diff
@@ -3012,6 +3012,45 @@ static int test_mod_exp_consttime(int i)
     return res;
 }
 
+/* Regression test for #17648. */
```

Comment:
Rather than reference a github issue I would rather describe what the test is about